nemo/collections/multimodal/parts/utils.py

```diff
@@ -321,7 +334,9 @@ def setup_trainer_and_model_for_inference(
         )
 
     else:
-        raise ValueError(f"Unrecognized checkpoint type: {cfg.model.restore_from_path}")
+        model = model_provider(cfg=cfg.model, trainer=trainer)
+        # raise ValueError(f"Unrecognized checkpoint type: {cfg.model.restore_from_path}")
```

Comment:
clean

Reply:
clean as in clean the comment or undo the change?